<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicAnalyser</class>
 <widget class="QDialog" name="LogicAnalyser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>385</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logic Analyser</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayoutNames">
       <property name="rightMargin">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="LogicView" name="graphicsViewLogicCurves">
       <property name="toolTip">
        <string>Displays the recorded pin states versus time.
The accuracy of time meaurement is affected by the
operating system sceduler
Pin states are:
- Binary value of an input or output pin
- Analog value of input pin editable in main list view
- Analog value of output pwm pin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelStatusbar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="toolTip">
        <string>Updates the view
Hint: May affect the accuracy of time during recording</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckAutoUpdate">
       <property name="toolTip">
        <string>Automatically updates view once a second
Hint: this may affect the accuracy of time</string>
       </property>
       <property name="text">
        <string>Auto Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRecord">
       <property name="toolTip">
        <string>Starts recording of pin states</string>
       </property>
       <property name="text">
        <string>Record</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="toolTip">
        <string>Clears the recorded pin states</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogicView</class>
   <extends>QGraphicsView</extends>
   <header location="global">logicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
